main.tf

```diff
@@ -222,7 +225,7 @@ module "monitoring" {
   location             = var.location
   node_count           = var.node_count
 
-  web_test_availability_request_url = module.application_gateway.public_ip_address_fqdn
+  web_test_availability_request_url = var.disable_agw ? null : module.application_gateway[0].public_ip_address_fqdn
```

Comment:
wouldn't this effectively disable the `web_test_availability_request`?

Reply:
Essentially yes. In Azure our VMs don't have public IP so we cannot use this if there is no application gateway.

Reply:
Well this would leave you with unconfigured availability test

Reply:
That is true, however we would have nothing to test against? Could think of a way to make the test use the new AGW perhaps?

Reply:
The idea of this test is to check if the deployment is alive, if external(already existing) API Gateway, therefore an external domain should be provided, I'm not seeing such in your changes.

Reply:
This is still valid

Reply:
I have fixed this. It now works with the external gateway.